nginx-proxy/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM nginx
+
+RUN apt-get update && apt-get install -y netcat
+
+COPY entrypoint.sh /opt/entrypoint.sh
+RUN chmod +x /opt/entrypoint.sh
+
+COPY default.conf /etc/nginx/default.conf.template
+
+ENTRYPOINT ["/opt/entrypoint.sh"]
+
+CMD ["nginx", "-g", "daemon off;"]
```

Comment:
Will this be changed to `daemon on` at some point? Or is the idea to have this container just running `nginx` so it should be in the foreground all the time?

Reply:
It should run as a foreground process in a container. The container should basically run a single process, so there is no need for it to run in the background.